### ISSUE TITLE
Flash of text when creating tabs with a status bar plugin

### ISSUE THREAD

**Issue by oredaze** [zellij-org/zellij#3082]
I am using a plugin for the status bar called [zjstatus](https://github.com/dj95/zjstatus)

When creating new tabs (often, but not always) there will be a very quick flash of text behind the tabs that says something like:
`Loading from filepath/zjstatus.wasm`
The plugin seems to load on each new tab creation.

[The issue](https://github.com/dj95/zjstatus/issues/33) I have opened on their page concluded with the recommendation to ask here if something can be done. They say it likely is not a problem on their end.

Is there a way for zellij to cache the plugin or something, such that it speeds up loading / doesn't need to load each time?

**Comment by imsnif** [comment]
Plugins are cached by default, and this flash of text should be very quick. Could you post a video of this, so that I can understand if this is a "by design" flash or a "my cache isn't working" flash?

**Comment by oredaze** [comment]
https://github.com/zellij-org/zellij/assets/51076873/5f4da338-a294-44b5-810d-59051fdd70a6

**Comment by imsnif** [comment]
Yeah, this is a by design thing I'm afraid as the plugin is already cached (otherwise it would have been much longer). We might solve it in the future by only showing the loading info after 1 second or some such.
